build.gradle

```diff
@@ -1,36 +1,78 @@
 plugins {
+    id 'org.springframework.boot' version '3.0.0-M2'
```

Comment:
Why did you choose for the 3.0.0 milestone version of the spring boot plugin and the 2.x version of Spring Boot itself?